service/s3/marshal.go

```diff
@@ -112,23 +112,23 @@ func marshalPutObjectInputShapeAWSREST(input *PutObjectInput, encoder *rest.Enco
 	}
 	if input.Expires != nil {
 		if err := encoder.AddHeader("Expires").Time(*input.Expires, protocol.RFC822TimeFormatName); err != nil {
-			return awserr.New(aws.ErrCodeSerialization, "failed to marshal header for shape Expires using REST encoder:\n \t %v", err)
+			return awserr.New(aws.ErrCodeSerialization, "failed to marshal header for shape Expires using REST encoder: %v", err)
 		}
 	}
 	if input.ObjectLockRetainUntilDate != nil {
 		if err := encoder.AddHeader("x-amz-object-lock-retain-until-date").Time(*input.ObjectLockRetainUntilDate, protocol.ISO8601TimeFormatName); err != nil {
-			return awserr.New(aws.ErrCodeSerialization, "failed to marshal header for shape ObjectLockRetainUntilDate using REST encoder:\n \t %v", err)
+			return awserr.New(aws.ErrCodeSerialization, "failed to marshal header for shape ObjectLockRetainUntilDate using REST encoder: %v", err)
```

Comment:
This error message shouldn't include the trailing `%v` this component will be added automatically by the SDK's `awserr.Error` type.